packages/api-tests/test/browser-test-setup.js

```diff
@@ -0,0 +1,70 @@
+const webdriverio = require('webdriverio');
+const selenium = require('selenium-standalone');
+
```

Comment:
Looks good, nice to see that the current specs can be run in the browser without much setup 👍 

Will the browser tests run on CI?